src/main/java/api/models/response/CrawlingJobData.java

```diff
@@ -0,0 +1,33 @@
+package api.models.response;
+
+public class CrawlingJobData  {
```

Comment:
could it be that we need to update the values after some steps?

Reply:
The intention of class is to be instantiated once per request put all data together and return it as a http response. It will be another class that can be used to be update values after some steps